packages/ansible.rb

```diff
@@ -1,32 +1,23 @@
 require 'package'
 
 class Ansible < Package
-  description 'Ansible is a radically simple IT automation engine that automates cloud provisioning, configuration management, application deployment, intra-service orchestration, and many other IT needs.'
+  description 'Ansible is a radically simple IT automation platform from RedHat.'
```

Comment:
Why do we need to shorten the description?  For someone searching for certain keywords, it would be helpful to keep it as is.

Reply:
alright, i can change it back